src/main.js

```diff
@@ -1,14 +1,42 @@
 import Vue from 'vue';
+import '@firebase/auth';
+import firebase from 'firebase/app';
 import App from './App.vue';
 import './registerServiceWorker';
 import router from './router';
-import store from './store';
+import store from './store';// eslint-disable-line import/no-cycle
 import vuetify from './plugins/vuetify';
 import 'roboto-fontface/css/roboto/roboto-fontface.css';
-// import '@mdi/font/css/materialdesignicons.css';
 
 Vue.config.productionTip = false;
 
+require('dotenv').config();
+
+const firebaseConfig = {
+  apiKey: process.env.VUE_APP_API_KEY,
+  authDomain: process.env.authDomain,
+  databaseURL: process.env.VUE_APP_DATABASE_URL,
+  projectId: 'questlists',
+  storageBucket: process.env.VUE_APP_STORAGE_BUCKET,
+  messagingSenderId: process.env.messagingSenderId,
+  appId: process.env.appId,
+};
+
+export const FBApp = firebase.initializeApp(firebaseConfig);
+
+export const auth = FBApp.auth();
+// user is looking at to the firebase.auth().currentUser
+// no need to redefine it
+auth.onAuthStateChanged((user) => {
+  if (user) {
+    // User is signed in
```

Comment:
comments should show why, not what

Reply:
I feel you are asking about the User is signed in comment to change it to check if a user is signed in or not?

Reply:
Yes. This comment is obvious by the code - there is no need to have it.

Reply:
Extraneous comments make the code harder to read. Also, there is the issue of what happens to the comment when the code is changed. Maybe in the future, someone will change the code, or the API will change. Then the person who changes it will have to change the comment. This is basically adding more cognitive load to future developers, not taking from it. Your code should be as declarative and self-documenting as possible.